### PR TITLE
feat: merge national subnational[HE-514]

### PR DESCRIPTION
**Pull request by anamontiaga**
### Feature relevant tickets
[HE-514](https://vizzuality.atlassian.net/browse/HE-514)

### CODE REVIEW

File: src/constants/analyze-areas-constants.js [hunk 1]
```diff
@@ -90,10 +91,7 @@ export const getSpeciesFilters = () => [
 ]
 
 export const PRECALCULATED_LAYERS_CONFIG = {
-  [GADM_0_ADMIN_AREAS_FEATURE_LAYER]: {
-    name: 'NAME_0',
-  },
-  [GADM_1_ADMIN_AREAS_FEATURE_LAYER]: {
+  [ADMIN_AREAS_FEATURE_LAYER]: {
     name: 'NAME_1',
```
Comment (Bluesmile82) [inline src/constants/analyze-areas-constants.js:95]: How is this working when we select gadm_0? 🤔

File: src/constants/map-tooltip-constants.js [hunk 2]
```diff
@@ -9,15 +9,15 @@ import {
...
-    title: 'NAME_0',
-    id: 'MOL_ID'
-  },
-  [GADM_1_ADMIN_AREAS_FEATURE_LAYER] : {
+  [ADMIN_AREAS_FEATURE_LAYER] : {
     title: 'NAME_1',
     subtitle: 'NAME_0',
     id: 'MOL_ID'
   },
+  [GADM_0_ADMIN_AREAS_FEATURE_LAYER] : {
```
Comment (Bluesmile82) [inline src/constants/map-tooltip-constants.js:17]: Why do we only have gadm_0 here and no gadm_1?

File: src/constants/mol-layers-configs.js [hunk 3]
```diff
@@ -138,19 +138,19 @@ export const layersConfig = {
...
     type: LAYER_TYPES.FEATURE_LAYER,
-    url: LAYERS_URLS[GADM_0_ADMIN_AREAS_FEATURE_LAYER],
+    url: LAYERS_URLS[ADMIN_AREAS_FEATURE_LAYER],
     renderer: BORDERS_LAYERS_RENDERER,
     bbox: null
   },
-  [GADM_1_ADMIN_AREAS_FEATURE_LAYER]: {
-    title: GADM_1_ADMIN_AREAS_FEATURE_LAYER,
-    slug: GADM_1_ADMIN_AREAS_FEATURE_LAYER,
+  [GADM_0_ADMIN_AREAS_FEATURE_LAYER]: {
```
Comment (Bluesmile82) [inline src/constants/mol-layers-configs.js:149]: Same here, why only gadm_0?

File: src/constants/search-location-constants.js [hunk 4]
```diff
@@ -10,6 +11,13 @@ import {
 import { LAYERS_URLS } from 'constants/layers-urls';
 
 export const SEARCH_SOURCES_CONFIG = {
+  [ADMIN_AREAS_FEATURE_LAYER]: {
```
Comment (Bluesmile82) [inline src/constants/search-location-constants.js:14]: We will check this on the next feature

File: src/containers/scenes/aoi-scene/aoi-scene-utils.js [hunk 5]
```diff
@@ -97,11 +97,12 @@ export const setPrecalculatedAOIs = ({ areaTypeSelected, precalculatedLayerSlug,
     }
 
     const setNationalOrSubnationalType = () => {
+      console.log('hello', attributes)
```
Comment (Bluesmile82) [inline src/containers/scenes/aoi-scene/aoi-scene-utils.js:100]: Please remove this

File: src/containers/scenes/data-scene/data-scene.js [hunk 6]
```diff
@@ -76,8 +75,19 @@ const Container = (props) => {
...
-    precomputedAoiAnalytics(mapTooltipContent.title);
-    browsePage({ type: AREA_OF_INTEREST, payload: { id: mapTooltipContent.id }, query: { precalculatedLayer: selectedAnalysisLayer.slug, OBJECTID: mapTooltipContent.objectId } });
+    const { title } = mapTooltipContent;
+    const { isSubnational } = mapTooltipData;
+    precomputedAoiAnalytics(title);
+
+    const isAdminNational = areaTypeSelected === AREA_TYPES.administrative && !isSubnational;
+    const isAdminSubnational = areaTypeSelected === AREA_TYPES.administrative && isSubnational;
+
+    const precalculatedLayer = () => {
```
Comment (Bluesmile82) [inline src/containers/scenes/data-scene/data-scene.js:85]: We have to remember to refactor this

File: src/utils/geo-processing-services.js [hunk 7]
```diff
@@ -203,9 +201,10 @@ export const setPrecalculatedSpeciesData = (attributes, setTaxaData) => {
   getPrecalculatedSpeciesData(AMPHIBIANS, attributes.amphibians).then(data => setTaxaData(data));
 };
 
-const getAreaName = (data, config) => {
-  if (!config) return null;
-  return config.subtitle ? `${data[config.name]}, (${data[config.subtitle]})` : data[config.name];
+const getAreaName = (data) => {
```
Comment (Bluesmile82) [inline src/utils/geo-processing-services.js:204]: I see that we are overriding the config here and maybe that is why it doesn't matter what we have there

File: src/utils/map-tooltip-service.js [hunk 8]
```diff
@@ -21,7 +21,7 @@ export const setMapTooltipData = ({  molId, setLandVertebrateSpeciesNum, setProt
 
     setProtectedAreaTooltipData({
       description: `${attributes.DESIG}, ${attributes.STATUS.toLowerCase()} in ${attributes.STATUS_}`,
-      status: attributes.STATUS_,
+      status: attributes.STATUS,
```
Comment (Bluesmile82) [inline src/utils/map-tooltip-service.js:24]: 👍

**Review by Bluesmile82 — CHANGES_REQUESTED** [review]
Great work! Please take a look at some inconsistencies declaring gadm_0 and gadm_1 layers

File: src/containers/scenes/data-scene/data-scene.js (hunk 6 above)
Comment (anamontiaga) [reply src/containers/scenes/data-scene/data-scene.js:85]: Added a subtask [HE-519](https://vizzuality.atlassian.net/browse/HE-519)

File: src/utils/geo-processing-services.js (hunk 7 above)
Comment (anamontiaga) [reply src/utils/geo-processing-services.js:204]: Added protected area case

**Comment by anamontiaga** [comment]
There is still a bug that I haven't been able to fix yet. On some occasions, the passage to the aoi scene fails, because we are not obtaining whether it is national or subnational admin area type when setPrecalculatedAOIs

**Pull request closed without merging**